src/navigate-pop/switching-apps.md

```diff
@@ -0,0 +1,22 @@
+# Navigate Between Running Applications
+
+Pressing SUPER + Tab will display a list of running application icons across the screen. Continue holding SUPER and pressing Tab to toggle between applications
+
+![Toggle Apps](/images/locate-launch-applications/toggle-apps.png)
+
+Mouse over or release tab on highlighted application icons to display multiple open windows for that application. Continue holding SUPER and press ` to toggle between the multiple windows.
```

Comment:
Is there a reason to not mention using the left and right arrows as well for switching between multiple windows?

Reply:
This page seems to be talking about switching between apps and not windows, although it might make sense to cover both in the same place so people can better understand the differences. The lack of Alt-Tab (for window switching) in comparison to Super-Tab is also a little odd.

Reply:
In additional alt-tab mention, I've added a window navigation section to this page.

Reply:
Resolving unless further review suggests a change related to this comment.